examples/cli/README.md

```diff
@@ -1,22 +1,89 @@
-# XLI
+# CLI
 
-XLI is an demo XMTPv3 console client, which allows developers to send and receive messages via the command line. While potentially useful its primary purpose is to demonstrate how to make a client application using the Rust Apis.
+![Status](https://img.shields.io/badge/Project_status-Alpha-orange)
 
-## Running
+This is a demo XMTP v3-alpha console client (CLI) that you can use to send and receive messages via the command line. Specifically, you can use it to try out [double ratchet messaging](#double-ratchet-messaging) and [installation key bundles](#installation-key-bundles) enabled by XMTP v3-alpha.
 
-### Register accounts
+> **Important**  
+> This software is in **alpha** status and ready for you to start experimenting with. Expect frequent changes as we add features and iterate based on feedback.
 
-`./xli.sh --db user1.db3 register`
-`./xli.sh --db user2.db3 register`
+## Send a double ratchet message
 
-### Get wallet address
+Use the CLI to send a [double ratchet message](#double-ratchet-messaging) between test wallets using a local XMTP node and database running in Docker Desktop.
 
-`./xli.sh --db user2.db3 info`
+1. In the root directory of this repo, run:
 
-### Send message
+   ```bash
+   dev/up
+   ```
 
-`./xli.sh --db user1.db3 send <user2_address> "hello"`
+2. Create a sender wallet account (user1). Create an [installation key bundle](#installation-key-bundles) and store it in the database. Grant the installation key bundle permission to message on behalf of the sender address. This will allow the CLI to message on behalf of the sender address.
```

Comment:
Sorry, I thought this part of the document was going to be in the `examples/cli` folder. We should explicitly tell people to move to that folder between step 1 and 2, since the commands from 2 onwards will only work in that folder.

Reply:
Such a great callout @neekolas - thank you! The README is in `examples/cli`, but the `dev/up` command needs to be run at the root. So absolutely important to make going to `examples/cli` an explicit step.

Reply:
We no longer need to run `dev/up`, because the CLI points to the dev network by default, which allows messaging across multiple machines.

If people explicitly want to run against localhost, they can run `dev/up` and then specify the `--local` flag on the CLI.

Reply:
Thank you @richardhuaaa! Updating now..... <3